transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/PackageModule.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2021 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.plugin;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+
+/**
+ * Represent a package module in a dependency
+ */
+public class PackageModule {
+  private static final String GROUP_KEY = "group";
+  private static final String MODULE_KEY = "module";
+  private final String _group;
+  private final String _module;
+
+  public PackageModule(String group, String module) {
+    this._group = group;
+    this._module = module;
+  }
```

Comment:
Why public constructor and `of` method at the same time. Let us just use the constructor? You can add a constructor for the group only and remove both `of` methods.

Reply:
will remove based on your comment to remove PackageModul